event-types.json

```diff
@@ -25,6 +25,33 @@
         }
       }
     },
+    "blood pressure": {
+      "description": "To record a blood pressure measurement.",
+      "formats": {
+        "mmhg-bpm": {
+          "description": "The blood pressure measurement. Pressures are stored in mmHg and the heart rate in bpm",
+          "type": "object",
+          "properties": {
+            "systolic": {
+              "type": "number",
+              "description": "Systolic pressure. Unit: mmHg."
```

Comment:
The field order should be consistent across the file (in this case: `description` first) (here and below)